src/azure-cli/azure/cli/command_modules/sqlvm/_validators.py

```diff
@@ -108,3 +108,19 @@ def validate_sqlmanagement(namespace):
 
     if (sql_mgmt_mode == "NoAgent" and (namespace.sql_image_sku is None or namespace.sql_image_offer is None)):
         raise CLIError("usage error: --sql-mgmt-type NoAgent --image-sku NAME --image-offer NAME")
+
+
+# pylint: disable=too-many-statements,line-too-long
+def validate_expand(namespace):
+    '''
+    Validates if expand passes accepted values.
+    '''
+    allowed_values = ['*', 'AutoBackupSettings', 'AutoPatchingSettings', 'KeyVaultCredentialSettings', 'ServerConfigurationsManagementSettings']
+
+    if namespace.expand is not None:
+        items = namespace.expand.split(",")
+        if "*" not in items:
```

Comment:
>"*" [](start = 11, length = 3)

what happens if customer passes something like *,Test,Mess -> What will be the behavior?

Reply:
Since * is included, the * will override the unacceptable values (Test, Mess) and display all the settings. Same goes for a case like Test,*,Mess.

Reply:
Ok cool

---
In reply to: [488296563](https://github.com/Azure/azure-cli/pull/15156#discussion_r488296563) [](ancestors = 488296563)